synthetic-monitoring/README.md

```diff
@@ -0,0 +1,44 @@
+# Synthetic Monitoring Jsonnet Library
+
+This library simplifies interaction with the Grafana Synthetic Monitoring API.
```

Comment:
This reads like the library interacts with the API. Can you rephrase? Like 'generates objects/checks for SM' or something along the lines of that.